forge-gui/res/cardsfolder/upcoming/blood_money.txt

```diff
@@ -0,0 +1,9 @@
+Name:Blood Money
+ManaCost:5 B B
+Types:Sorcery
+A:SP$ DestroyAll | ValidCards$ Creature | RememberDestroyed$ True | SubAbility$ DBTreasure | SpellDescription$ Destroy all creatures.
+SVar:DBTreasure:DB$ Token | TokenAmount$ X | TokenTapped$ True | TokenScript$ c_a_treasure_sac | TokenOwner$ You | SubAbility$ DBCleanupSpellDescription$ For each nontoken creature destroyed this way, create a tapped Treasure token.
```

Comment:
Can't believe this script is working with `SubAbility$ DBCleanupSpellDescription$` !!!

Reply:
Ah I guess it just wouldn't do the cleanup